.github/scripts/ci/find_broken_wikilinks.py

```diff
@@ -0,0 +1,173 @@
+import os
+import sys
+
+def red(s):
+    return f"\x1b[31m{s}\x1b[0m"
+
+def green(s):
+    return f"\x1b[32m{s}\x1b[0m"
+
+def yellow(s):
+    return f"\x1b[33m{s}\x1b[0m"
+
+def blue(s):
+    return f"\x1b[34m{s}\x1b[0m"
+
+redirects = {}
+with open("wiki/redirect.yaml", 'r', encoding='utf-8') as file:
+    content = file.read()
+    for linenumber, line in enumerate(content.split('\n'), start=1):
+        split = line.split('"')
+        try:
+            redirects[split[1]] = (split[3], linenumber)
+        except Exception:
+            pass
+
+def child(path):
+    return path[path.find('/', 1) + 1:]
+
+def directory(filename):
+    return filename[filename.find('/') + 1:filename.rfind('/')]
+
+note = ""
+def check_redirect(link):
+    global note
+    link = link.lower()
+    try:
+        redirect = redirects[link]
+    except KeyError:
+        return False
+    if not os.path.exists(f"wiki/{redirect[0]}"):
+        note = f"{blue('Note:')} Broken redirect (redirect.yaml:{redirect[1]}: {link} --> {redirect[0]})"
+        return False
+    return True
+
+def check_link(directory, link):
+    global note
+    note = ""
+    if link.startswith("/wiki/"):
+        # absolute wikilink
+        if os.path.exists(link[1:]):
+            return True
+        else:
+            # may have a redirect
+            return check_redirect(child(link))
+    elif not (link.startswith("http") or link.startswith("mailto:") or link.startswith("irc:")):
```

Comment:
to protect against `httpp://` and the likes, I'd use `://` for HTTP(S) (also, I don't think we need `irc:`)

```python
elif not any(link.startswith(prefix) for prefix in ("http://", "https://", "irc:", "mailto:"))
```

Reply:
ok i've identified the only `irc:` links as being in outdated translations of `Community/Internet_Relay_Chat`